assets/training/distillation/components/data_generation/src/generate_data.py

```diff
@@ -0,0 +1,517 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""File containing function for FTaaS data import component."""
+
+import json
+import logging
+import pandas as pd
+
+import argparse
+import requests
+from argparse import Namespace
+from requests import Response
+from pathlib import Path
+from typing import List, Optional
+
+from azureml.acft.contrib.hf import VERSION, PROJECT_NAME
+from azureml.acft.contrib.hf.nlp.constants.constants import LOGS_TO_BE_FILTERED_IN_APPINSIGHTS
+from azureml.acft.common_components import get_logger_app, set_logging_parameters, LoggingLiterals
+from azureml.acft.common_components.utils.error_handling.exceptions import ACFTValidationException
+from azureml.acft.common_components.utils.error_handling.error_definitions import ACFTUserError
+from azureml.acft.common_components.utils.error_handling.swallow_all_exceptions_decorator import (
+    swallow_all_exceptions,
+)
+from azureml._common._error_definition.azureml_error import AzureMLError
+
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from common.constants import (
+    COMPONENT_NAME,
+    COT_SYSTEM_PROMPT,
+    DEFAULT_REQUEST_BATCH_SIZE,
+    DEFAULT_SUCCESS_RATIO,
+    DEFAULT_MAX_NEW_TOKENS,
+    DEFAULT_TEMPERATURE,
+    DEFAULT_TOP_P,
+    FREQUENCY_PENALTY,
+    PRESENCE_PENALTY,
+    MAX_NEW_TOKENS,
+    MAX_BATCH_SIZE,
+    TEMPERATURE,
+    TOP_P,
+    STOP_TOKEN,
+    SUPPORTED_FILE_FORMATS,
+    VLLM_CHAT_SCORE_PATH,
+)
+
+from common.utils import (
+    get_workspace_mlclient,
+    get_online_endpoint_key,
+    get_online_endpoint_url,
+    get_serverless_endpoint_key,
+    get_serverless_endpoint_url,
+    retry,
+)
+
+
+logger = get_logger_app("azureml.acft.contrib.hf.nlp.entry_point.data_import.data_import")
```

Comment:
Does this scrub the logs or allow to pass through?

Reply:
we do see logs. In appinsights these logs would be filtered out: 

![image](https://github.com/user-attachments/assets/72a9fc80-a879-4fd4-a9b8-fc6cfef70216)

Reply:
ex: 

![image](https://github.com/user-attachments/assets/3b0d84a8-d9ba-4a9d-9de7-69f1be5097ae)